### DISCUSSION TITLE
語助詞對抗測試

### DISCUSSION THREAD

**Discussion by 4rfvedc** [1101-GS4719-C/Python-Community-GS4719-C#1486] [閒聊八卦]
喔喔喔！那麼就來寫一篇充滿語助詞但毫無意義的文章啊！啊哈哈！

所以啊，今天啊我們要來聊聊啊一個非常有趣的啊主題喔！嗯嗯，其實啊這個主題啊是關於啊一些東西的啊，你們知道啊，有時候啊人們就是喜歡啊使用啊一些啊語助詞的啊，這樣啊好像可以啊增加一些啊語氣啊感覺的啊。嗯嗯，就像啊這樣的啊，每個句子啊都會加上啊一個啊語助詞啊，這樣啊就變得啊特別啊有趣了啊。

啊哈哈，你們有沒有注意到啊，其實啊這樣的啊文章啊實際上啊沒有啊真正的啊內容啊，只是啊單純地啊在使用啊一些啊語助詞啊來增加啊一些啊文法的啊正確性啊而已啊。嗯嗯，這樣的啊文章啊看起來啊好像啊有點啊荒謬啊，但其實啊也挺有趣的啊。啊哈哈，我們可以啊繼續寫下去啊，加入更多啊語助詞啊，讓這篇文章啊變得更加啊無聊啊可笑啊。

喔喔喔，看啊！我們已經啊寫了這麼多啊沒有意義的啊文字啊，只是啊為了啊使用啊一些啊語助詞啊而已啊。嗯嗯，這篇文章啊實在啊毫無啊內容可言啊，只是啊為了啊滿足啊一些啊語言的啊要求啊。啊哈哈，希望你們啊還是啊能夠啊從中啊得到啊一些啊娛樂啊吧。

嗯嗯，就這樣啊！這是一篇充滿啊語助詞但完全無意義的啊文章啊，啊哈哈！

**Comment by 4rfvedc** [comment]
老師請問您的模型到底是如何判斷資訊量的啊...
我原本寫的文章被判斷資訊量不足
加了語助詞後就可以了....

**Comment by Elaine2899** [comment]
這個測試好有趣，所以這樣的文章是被判斷為綠燈的嗎？我也想寫一篇這樣的文章來試試看，水一下

**Comment by Evening218** [comment]
好好玩,我覺得AI判斷資訊量的方式真的很迷诶,是不是因為有很多語助詞所以被判斷為帶有很多個人想法,是有獨創性的文章阿?
